locales/ar-arb/messages.po

```diff
@@ -251,7 +254,7 @@ msgid "The identifier of the choice. This identifier must not be used by any oth
 msgstr "معرّف الاختيار. يجب عدم استخدام هذا المعرف من قبل أي استجابة أخرى أو متغير بند. المعرّف هو سلسلة من الأحرف التي يجب أن تبدأ بحرف أو شرطة سفلية (\"_\") وتحتوي فقط على أحرف ، وشرطات سفلية ، وواصلات (\"-\") ، ونقطة (\".\" ، ويعرف أيضًا باسم النقطة الكاملة) ، وأرقام ، والجمع بين الأحرف والموسعات."
 
 msgid "There is already one interaction with the same identifier \"%s\" (label : \"%s\") and same version : %s. Do you want to override the existing one ?"
-msgstr "يوجد بالفعل تفاعل واحد مع نفس المعرف \"٪ s\" (التسمية: \"٪ s\") ونفس الإصدار:٪ s. هل تريد تجاوز الموجود؟"
+msgstr "يوجد بالفعل تفاعل واحد مع نفس المعرف  \"%s\" (التسمية: \"%s\") ونفس الصيغة: %s . هل تريد تجاوز الموجود?"
```

Comment:
Are you absolutely sure about that question mark?
I'm no expert in Arabic, but Google puts the question mark on the left (at the end of the sentence).
https://translate.google.com/?sl=auto&tl=ar&text=Do%20you%20want%20to%20override%20the%20existing%20one%3F&op=translate

Let's re-check it with a help of a PM.

Same about the full stop mark in the first sentence (I know it's like that in the original copy, though).

Reply:
Not at all. These translations were provided by the customer. What you said makes sense, perhaps we can revert back to the Arabian question mark symbol. This is how it looks at the moment:

![image](https://user-images.githubusercontent.com/1084316/152953662-90cd78b8-52f2-4077-8704-3719724032da.png)

Reply:
Fixed: 
![image](https://user-images.githubusercontent.com/1084316/153014472-ba14b3ed-a942-4263-9d57-d5b7222b9abf.png)